src/helpers/file.js

```diff
@@ -25,15 +25,26 @@ exports.fileToBase64 = filePath => new Promise((resolve, reject) => {
   fileStream.once('error', err => reject(err));
 });
 
-exports.urlToBase64 = async (url) => {
+exports.downloadImages = async (imageList) => {
+  if (!fs.existsSync('src/data/tmp')) {
+    fs.mkdirSync('src/data/tmp');
+  }
   try {
-    const image = await axios.get(url, { responseType: 'arraybuffer' });
-    const raw = Buffer.from(image.data).toString('base64');
-
-    return `data:${image.headers['content-type']};base64,${raw}`;
+    imageList.forEach(async (image) => {
```

Comment:
Il me semble avoir testé il n'y a pas longtemps des await dans les forEach et cela ne fonctionne pas. Essaye avec une boucle for pour voir si ca resoud ton probleme :)

Reply:
done ça marche au final top !